drivers/clock_control/clock_stm32_ll_common.c

```diff
@@ -253,6 +253,8 @@ static inline int stm32_clock_control_configure(const struct device *dev,
 		return 0;
 	}
 
+	sys_clear_bits(DT_REG_ADDR(DT_NODELABEL(rcc)) + STM32_CLOCK_REG_GET(pclken->enr),
+		       STM32_CLOCK_MASK_GET(pclken->enr) << STM32_CLOCK_SHIFT_GET(pclken->enr));
```

Comment:
Question: instead of having two calls, could we use the MODIFY_REG macro (specific to ST)? Or create a function that does the same job in sys_bitops.h, like sys_modify_bits?

Reply:
That is not related to this PR. This is non blocking

Reply:
Completely agrees that it is not blocking. I was just wondering if a more elegant solution could be introduced, as there are probably other places where it could be useful.